res/bnd/tomcat-embed-core.jar.tmp.bnd

```diff
@@ -131,15 +131,15 @@ Provide-Capability: \
         access=32;\
         version=${Bundle-Version};\
         modules='\
-            jakarta.ejb-api,\
-            jakarta.mail,\
-            jakarta.persistence-api,\
-            jakarta.xml.rpc-api,\
-            java.xml.ws'
+            javax.ejb.api,\
+            javax.mail,\
+            javax.persistence.api,\
+            javax.xml.rpc.api,\
+            javax.xml.ws.api'
 -jpms-module-info-options: \
     ${module.name};substitute=tomcat-embed-core,\
-    jakarta.ejb-api;static=true,\
-    jakarta.mail;static=true,\
-    jakarta.persistence-api;static=true,\
-    jakarta.xml.rpc-api;static=true,\
-    java.xml.ws;static=true
+    javax.ejb.api;static=true,\
+    javax.mail;static=true,\
+    javax.persistence.api;static=true,\
```

Comment:
This looks to have a module name of `java.persistence` in Java EE 8

Reply:
fixed